frontend/src/components/institute/EditSchoolContactPage.vue

```diff
@@ -0,0 +1,290 @@
+<template>
+  <v-card
+      id="editSchoolContactCard">
+    <v-card-title class="sheetHeader pt-1 pb-1">Edit Contact</v-card-title>
+    <v-divider></v-divider>
+    <v-card-text>
+      <v-form ref="editSchoolContactForm" v-model="isFormValid">
+        <v-row class="d-flex justify-center">
+          <v-col>
+            <v-select
+                id='schoolContactTypeDropdown'
```

Comment:
not sure if you wanted editSchoolContactTypeDropdown to match the other ones below.